src/code/z_demo.c

```diff
@@ -405,7 +405,7 @@ void CutsceneCmd_Misc(PlayState* play, CutsceneContext* csCtx, CsCmdMisc* cmd) {
             break;
 
         case CS_MISC_SET_FLAG_LAKE_HYLIA_RESTORED:
-            SET_EVENTCHKINF(EVENTCHKINF_69);
+            SET_EVENTCHKINF(EVENTCHKINF_RESTORED_LAKE_HYLIA);
```

Comment:
Im fine with changing this to "restored lake hylia", but the cs misc command should probably be changed to match